code/game/machinery/washing_machine.dm

```diff
@@ -15,30 +15,48 @@
 	//8 = blood, running
 	//0 = closed
 	//1 = open
-	var/hacked = 1 //Bleh, screw hacking, let's have it hacked by default.
+	var/hacked = TRUE //Bleh, screw hacking, let's have it hacked by default.
 	//0 = not hacked
 	//1 = hacked
 	var/gibs_ready = 0
 	var/obj/crayon
 	var/speed_coefficient = 1
+	var/size_coefficient = 1
+	//Lists for each bin tier in order T1, T2, T3, T4. Whitelist doesn't change but its there for future proofing.
```

Comment:
maybe rework this like this:
if T1 check list entries T1-4
if T2 check list entries T2-4
etc

this would avoid having to c&p every obj in the other tiers if you added something to lets say T4
my explanation is a bit shit, so here is the example with the blacklist:

> list(/mob/living/carbon, /obj/item/clothing/head/helmet, /obj/item/clothing/head/syndicatefake, /obj/item/clothing/head/bomb_hood),\
list(/mob/living/carbon, /obj/item/clothing/suit/space, /obj/item/clothing/suit/syndicatefake, /obj/item/clothing/suit/cyborg_suit, /obj/item/clothing/suit/bomb_suit, /obj/item/clothing/suit/armor),\
list(/mob/living/carbon/human),\
list(/obj/item/clothing/mask/cigarette)\

Reply:
do robos solution, should be easier to add

but this copy&paste stuff is neither futureproof or good code